adoc/MAIN.release-notes.adoc

```diff
@@ -10,7 +10,7 @@ ifeval::['{release_type}' != 'public']
 endif::[]
 
 [WARNING]
-This software is not ready for production use.
+This software is not ready for productive use.
```

Comment:
Please change this back.
The software IS productive but can't be used in production. 

Production means stable software that companies roll out for their daily activities. A beta should never be used in production.